src/shared/mkfs-util.h

```diff
@@ -5,7 +5,10 @@
 
 #include "sd-id128.h"
 
-#include "strv.h"
+typedef enum MkfsFlags {
+        MKFS_QUIET                        = 1 << 0,  /* Suppress mkfs command output */
+        MKFS_DISCARD                      = 1 << 1,  /* Enable 'discard' mode on the filesystem */
```

Comment:
not that it really matters, but that's a lot of inner space. 

alignment around = is great, but maybe a bit less space, but of course, doesn't really matter

Reply:
This was copy-paste from an existing flags field (maybe the copy one?)

Happy to shrink it down, but I guess it will grow again at some point...